src/main/java/ryuzupluginchat/ryuzupluginchat/listener/OutdatedCommandCaptureListener.java

```diff
@@ -0,0 +1,32 @@
+package ryuzupluginchat.ryuzupluginchat.listener;
+
+import java.util.Arrays;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerCommandPreprocessEvent;
+import ryuzupluginchat.ryuzupluginchat.RyuZUPluginChat;
+
+@RequiredArgsConstructor
+public class OutdatedCommandCaptureListener implements Listener {
+
+  private final RyuZUPluginChat plugin;
+
+  private final List<String> commands = Arrays.asList("/tell", "/reply", "/t", "/r");
```

Comment:
```suggestion
  private final List<String> commands = Arrays.asList("/tell", "/reply", "/t", "/r", "/msg", "/message", "/m");
```
(https://github.com/ucchyocean/LunaChat/blob/master/src/main/resources/plugin.yml)